resources/views/users/following.blade.php

```diff
@@ -0,0 +1,45 @@
+@extends('layouts.app')
+
+@section('content')
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-md-8">
+        @foreach ($followings as $following)
+          <div class="card">
+            <div class="card-haeder p-3 w-100 d-flex">
+              @if(!empty($following->thumbnail))
+                <img src="/storage/user/{{ $following->thumbnail }}" class="rounded-circle" width="50" height="50">
+              @else
+                <img class="rounded-circle" width="50" height="50">
+              @endif
+                <div class="ml-2 d-flex flex-column">
+                  <a href="{{url('/mypage',$following->id)}}">{{ $following->name }}</a>
+                </div>
+                @if ($following->is_following($login_user->id))
+                  <div class="px-2">
+                    <span class="px-1 bg-secondary text-light">フォローされています</span>
+                  </div>
+                @endif
+                <div class="d-flex justify-content-end flex-grow-1">
+                  @if ($login_user->is_following($following->id))
+                    <form action="{{ route('unfollow',  ['user' => $following->id]) }}" method="POST">
+                      {{ csrf_field() }}
+                      {{ method_field('DELETE') }}
+                      <button type="submit" class="btn btn-danger">フォロー解除</button>
+                    </form>
+                  @elseif (!$login_user->id == $following->id)
```

Comment:
`でなかったら…` という条件の場合は !=を使った方が良いです
```suggestion
                  @elseif ($login_user->id != $following->id)
```